source/contracts/reporting/Market.sol

```diff
@@ -61,6 +62,8 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
 
     function initialize(IUniverse _universe, uint256 _endTime, uint256 _feePerEthInAttoEth, ICash _cash, address _designatedReporterAddress, address _creator, uint256 _numOutcomes, uint256 _numTicks) public payable beforeInitialized returns (bool _success) {
         endInitialization();
+        invalidOutcomeIndex = _numOutcomes;
+        _numOutcomes += 1; // The INVALID outcome is always last
```

Comment:
Out of curiosity, why always last instead of first?  It feels like things would be simpler if it was first because we could just have a constant for `INVALID_OUTCOME = 0` rather than having to do `_numOutcomes` (less clear).  Also, `0` _feels_ like a nice number for INVALID.

Reply:
Would also save on storage, since it appears you are storing `invalidOutcomeIndex` explicitly, rather than just using the already stored `numOutcomes` (which incidentally is a waste of gas).